### PR TITLE
dotCMS/core#23681 Let's show the whole path when select a folder in the host selector 

### PR DESCRIPTION
**Pull request by zulqarnainvd**
### Proposed Changes
* show full path on host selector field

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **
image|image

### CODE REVIEW

**Comment by wezell** [comment]
@zulqarnainvd This does not work as expected.  It only shows the parent folder and not the full path. Look at a file asset on starter under `/images/gallery/beach`

**Comment by zulqarnainvd** [comment]
> @zulqarnainvd This does not work as expected. It only shows the parent folder and not the full path. Look at a file asset on starter under `/images/gallery/beach`

Noted, let me check

File: dotCMS/src/main/webapp/html/css/dijit-dotcms/dotcms.css [hunk 1]
```diff
@@ -6504,7 +6504,7 @@ select[multiple]:hover {
 }
 
 .dotCMSHostFileFilteringSelect .dijitTextBox {
-  width: auto;
+  width: 100%;
```
Comment (fmontes) [inline dotCMS/src/main/webapp/html/css/dijit-dotcms/dotcms.css:6507]: Can you make it, not but around 50%?

**Review by fmontes — CHANGES_REQUESTED** [review]

File: dotCMS/src/main/webapp/html/css/dijit-dotcms/dotcms.css (hunk 1 above)
Comment (zulqarnainvd) [reply dotCMS/src/main/webapp/html/css/dijit-dotcms/dotcms.css:6507]: Noted

File: core-web/libs/dojo-theme/scss/backend/_dojo.scss [hunk 2]
```diff
@@ -142,7 +142,7 @@ select[multiple] {
 
 .dotCMSHostFileFilteringSelect {
     .dijitTextBox {
-        width: auto;
+        width: 100%;
```
[the remark this answers is not in the document]
Comment (zulqarnainvd) [reply core-web/libs/dojo-theme/scss/backend/_dojo.scss:145]: Noted

**Comment by zulqarnainvd** [comment]
image

Also need to add `parentPath` on this function `findHostFolder` because in edit mode it is not returning the `parentPath`
@fmontes

**Pull request closed without merging**